isc-dhcp-server

```diff
@@ -0,0 +1,21 @@
+# Defaults for isc-dhcp-server initscript
```

Comment:
we can do the same thing with this file. Leave the default system configuration as is, and add only one line to source our case-specific configuration

`. /home/pi/conf/isc-dhcp-server`

Reply:
okay so basically i have to comment out or delete the last line and then source our configuration file.

Reply:
don't comment or delete anything from the original default system file after installation.

just source our own configuration at the bottom of the file

`echo '. /home/pi/ourconf' >> /etc/default/isc-dhcp-server`

Reply:
we no longer need this file, correct?